readme.md

```diff
@@ -156,6 +159,25 @@ You can also add `np` to a custom script in `package.json`. This can be useful i
 }
 ```
 
+### Custom Tests
```

Comment:
```suggestion
### Custom tests
```

And I don't think "custom tests" properly describes what it is.

Reply:
Any suggestions on what else we could use? User Defined Tests may be a more accurate way of telling the user what it actually does.